server-spi/src/main/java/com/flipkart/varadhi/server/spi/authz/AuthorizationProvider.java

```diff
@@ -20,7 +22,12 @@ public Future<Boolean> init(ConfigFileResolver resolver, AuthorizationOptions au
 
         @Override
         public Future<Boolean> isAuthorized(UserContext userContext, ResourceAction action, String resource) {
-            return Future.succeededFuture(false);
+            return Future.succeededFuture(true);
+        }
+
+        @Override
+        public Future<Boolean> isSuperAdmin(UserContext userContext) {
```

Comment:
do we need this method? 

I assumed that we wanted to hide this aspect inside hte authProvider totally. isAuthorized can erturn true, if he is superAdmin.